src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -183,9 +177,9 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
         /**
          * Returns true if at least one field is edited.
```

Comment:
Need to change this

Reply:
Resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)